src/test/java/com/microsoft/sqlserver/testframework/DBStatement.java

```diff
@@ -51,6 +51,10 @@
         super(dbConnection, null, "statement");
     }
 
+    DBStatement(DBConnection parent, Statement internal, int type, int concurrency, int holdability) {
+        super(parent, null, "statement");
+    }
```

Comment:
this constructor can be removed as we are not using any of the new parameters passed